src/main/java/ca/concordia/soen6441/project/gameplay/CountryAssignment.java

```diff
@@ -8,11 +8,13 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.logging.Logger;
 
 /**
  * Handles the assignment of countries to players in the game.
  */
 public class CountryAssignment {
+    private static final Logger LOGGER = Logger.getLogger(CountryAssignment.class.getName());
```

Comment:
Why are we using Java's Logging utility when we had to create a logger for this build?  We should be using this instead, imho.  @marcchicoye?